### PR TITLE
update: nl -> eng

### PR DESCRIPTION
**Pull request by JasperSikkema**
translated the remaining dutch parts of the document

### CODE REVIEW

**Comment by JasperSikkema** [comment]
branch name is wrong its about the acceptation document

**Comment by JasperSikkema** [comment]
also added chapter 2 responsiblities

File: docs/acceptation_plan.md [hunk 1]
```diff
@@ -1,61 +1,67 @@
...
 
-## 1. Inleiding
+## 1. Introduction
 
-### 1.1 Doel 
+### 1.1 Goal
 During this project there will be worked on building an application for a monetary system, in this case a cryptocurrency. This system can be compared with other monetary systems like Bitcoin, Ethereum and other cryptocurrencies. A cryptocoin has several important requirements that it must meet in order for it to be used as a valid cryptocurrency. This acceptationplan is therefore set up to give a measurable basis for the testing and accepting of the delivered product. In this document the agreements regarding the testing and accepting of the delivered cryptocurrency. Also the responsibilities of all the involved parties are captured in this document.
 
-### 1.2 Referenties
+### 1.2 Refrences
```
Comment (YouriLieverdink) [inline docs/acceptation_plan.md:8]: References

File: docs/acceptation_plan.md [hunk 2]
```diff
@@ -1,61 +1,67 @@
...
 
-## 2. Verantwoordelijken
+## 2. Responsibilities
+
+| Role    | Responsibilities |
+|   ---   |      ---         |
+| Team    | The team has the responsibility to make the application with the requirements given by the client. They also have to make sure they fulfil the Non-functional requirements like security. They have to document the project so every stakeholder understands the project.|
+| Client  | The client has to give the right requirements and has to read the documentation to check if the team understands the project. |
+| Teacher | The teacher checks if the documents are complete and contain the right information |
```
Comment (YouriLieverdink) [inline docs/acceptation_plan.md:16]: Missing a `.`.

File: docs/acceptation_plan.md [hunk 3]
```diff
@@ -1,61 +1,67 @@
...
+### 3.3 Reliability
+A blockchain has to be available at all times to assure the data is not altered. A large part of the blockchain needs to be up to assure integerity. Because of this an update cannot be pushed. It is possible to gradually install updates. The versions will be saved with a vesion control service. So if theres an error in a version the other versions can be used.
 
-De betrouwbaarheid van de gegevens zelf zal komen met de grote van de blockchain. Een transactie is geaccepteerd als het door meer dan de helft van de blockchain goedgekeurt is. Als de blockchain groot genoeg is kunnen we er vanuit gaan dat een gevalideerde transactie betrouwbaar is.
+The Reliability of the data itself will come with the size of the blockchain. A transaction is accepted when more than 50 percent agrees that the transaction is valid. When the blockchain has a reasenable size we can assume that the transaction is reliable.
```
Comment (YouriLieverdink) [inline docs/acceptation_plan.md:37]: Reasonable

**Review by YouriLieverdink — CHANGES_REQUESTED** [review]
A small amount of spelling mistakes.

**Review by YouriLieverdink — APPROVED** [review]

**Pull request merged**